### DISCUSSION TITLE
Passing Metadata to LLM

### DISCUSSION THREAD

**Discussion by mikehudson2** [FlowiseAI/Flowise#2023] [Q&A]
We have websites scraped to Vectara so that each Vectara document contains a web page of text and its metadata, including that page's web address. When querying Vectara using the Conversational Retrieval QA Chain and ChatOpenAI (GPT4), matching chunks are currently sent to GPT4. Is it possible to **also** send GPT4 the url metadata of the page from which the chunks were extracted?

Or to put it more generally, within Flowise how can we send chunk metadata to GPT4?

**Comment by HenryHengZJ** [comment]
Hmm to send both the content and URL, I don't think there is a straight forward way to do this yet, but something worth exploring.

Perhaps we can use VectorStore To Document node -> Custom Function -> Prompt Template

**Reply by RicardoFernandez-UY** [reply]
Hi @HenryHengZJ  would you mind adding a little detail on how this would work ? 

> "Perhaps we can use VectorStore To Document node -> Custom Function -> Prompt Template"

Just to help understand what's on your mind. 
Thank you!!!
